Integrated-Windows-Auth-Flow/src/main/java/IntegratedWindowsAuthFlow.java

```diff
@@ -27,13 +24,19 @@ public static void main(String args[]) throws Exception {
     }
 
     private static IAuthenticationResult acquireTokenIntegratedWindowsAuth() throws Exception {
+        //Load properties file and set properties used throughout the sample
+        Properties properties = new Properties();
+        properties.load(new FileInputStream(Thread.currentThread().getContextClassLoader().getResource("").getPath() + "application.properties"));
+        String authority = properties.getProperty("AUTHORITY");
+        String accounts = properties.getProperty("ACCOUNT");
+        String clientId = properties.getProperty("CLIENT_ID");
+        String userName = properties.getProperty("USER_NAME");
+        Set<String> scope = Collections.singleton("");
```

Comment:
Since it's a set with just a single empty string and doesn't need to be configured/isn't mentioned in the README, I figured leaving it as it was would be fine. If it's in the properties file then it would be a blank property just like username and password, which might make a user think they need to configure it themselves

Reply:
The properties file now has the properties the user needs to configure for the sample (username, password, clientid), and the properties that in a 'real' situation could be configured (scope, authority) with a note saying they don't need to be configured for this sample to work.